cmd/dep/testdata/harness_tests/remove/specific/case2/testcase.json

```diff
@@ -2,6 +2,7 @@
   "commands": [
     ["remove", "github.com/not/used"]
   ],
+  "error-expected": "",
```

Comment:
Oh right, we only compare golden files, not the testcases. I don't think it's a bug to update testcase files when `-update` is specified, e.g. updating the final vendor contents or expected-error message. It just looks odd in this instance because it's not fixing the test, and is just futzing with the schema/formatting.